File: APIRollTheDice-Backend/src/main/java/com/example/APIRollTheDice/Model/DTO/ConversationDTO.java
package com.example.APIRollTheDice.Model.DTO;


import java.time.LocalDateTime;
import java.util.List;

public class ConversationDTO {
    private Long id;
    private List<Long> idParticipants;
    private LocalDateTime createdAt;
    private List<Long> IdMessages;

    public ConversationDTO() {
    }

    public ConversationDTO(Long id, List<Long> idParticipants, LocalDateTime createdAt, List<Long> idMessages) {
        this.id = id;
        this.idParticipants = idParticipants;
        this.createdAt = createdAt;
        IdMessages = idMessages;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public List<Long> getIdParticipants() {
        return idParticipants;
    }

    public void setIdParticipants(List<Long> idParticipants) {
        this.idParticipants = idParticipants;
    }

    public LocalDateTime getCreatedAt() {
        return createdAt;
    }

    public void setCreatedAt(LocalDateTime createdAt) {
        this.createdAt = createdAt;
    }

    public List<Long> getIdMessages() {
        return IdMessages;
    }

    public void setIdMessages(List<Long> idMessages) {
        IdMessages = idMessages;
    }
}

File: RollTheDice/Assets/_Project/API/Model/DTO/ConversationDTO.cs
using System;
using System.Collections.Generic;
namespace Assets._Project.API.Model.DTO
{
    public class ConversationDTO{
        public long Id { get; set; }
        public List<long> IdParticipants { get; set; }
        public DateTime CreatedAt { get; set; }
        public List<long> IdMessages { get; set; }

        public ConversationDTO(){}
        public ConversationDTO(long id,List<long> idPaticipant,DateTime createdAt, List<long> idMessage)
        {
            Id = id;
            IdParticipants = idPaticipant;
            CreatedAt = createdAt;
            IdMessages = idMessage;
        }

  }
}